### PR TITLE
Use transformer to get robot pose.

### PR DESCRIPTION
**Pull request by skasperski**
This changes the pose input from a plain RigidBodyState input port to a transformer block. Current usage should not be affected too much, but the port's name changes from robot_pose to dynamic_transformations and frame names have to be set according to the system setup. Also some testing and feedback would be appreciated before merging this.

### CODE REVIEW

File: tasks/Task.cpp [hunk 1]
```diff
@@ -70,21 +70,16 @@ void Task::updateHook()
     motionCommand.rotation    = 0;
     motionCommand.heading     = 0;
 
-    if( _robot_pose.readNewest( rbpose ) == RTT::NoData)
+    Eigen::Affine3d robot2map;
+    if(!_robot2map.get(base::Time::now(), robot2map, false))
```
Comment (jmachowinski) [inline tasks/Task.cpp:74]: You are interested in the latest transformation here, using base::Time::now does not grantee, 
that you get a pose back. Better, register a callback for the transformation, and save the given value
to a class member.

Comment (skasperski) [reply tasks/Task.cpp:74]: I'm calling robot2map.get() with interpoalation = false, I thought this has the same effect, e.g. it returns the latest received transformation. In what situation will it give no result?

Comment (jmachowinski) [reply tasks/Task.cpp:74]: You are right, I overlooked the interpolation = false, in this case it will work as intended.

Comment (skasperski) [reply tasks/Task.cpp:74]: Ah okay, thanks for clarification!

**Pull request merged**